packages/plugin-commands-env/src/node.ts

```diff
@@ -115,14 +120,14 @@ async function downloadAndUnpackZip (
   await fs.promises.unlink(tmp)
 }
 
-function getNodeJSTarball (nodeVersion: string, releaseDir: string) {
+function getNodeJSTarball (nodeVersion: string, releaseDir: string, nodeMirror: string) {
   const platform = process.platform === 'win32' ? 'win' : process.platform
   const arch = normalizeArch(process.platform, process.arch)
   const extension = platform === 'win' ? 'zip' : 'tar.gz'
   const pkgName = `node-v${nodeVersion}-${platform}-${arch}`
   return {
     pkgName,
-    tarball: `https://nodejs.org/download/${releaseDir}/v${nodeVersion}/${pkgName}.${extension}`,
+    tarball: `${nodeMirror}/${releaseDir}/v${nodeVersion}/${pkgName}.${extension}`,
```

Comment:
what if nodeMirror will be set with a trailing slash?

Reply:
Happy to normalize this. Does tacking it on if it's missing seem enough?

```ts
function normalizeNodeMirrorOption(nodeMirror: string) {
  return nodeMirror.endsWith("/") ? nodeMirror : `${nodeMirror}/`;
}
```

Reply:
yes